### DISCUSSION TITLE
how to install

### DISCUSSION THREAD

**Discussion by Damokoez** [notmarek/BeFake#113] [Q&A]
Hello, I don't know how to use python and I don't know how to install BeFake, someone could help me please

**Comment by s-alad** [comment]
Hey, I fixed my web client that allows you to use most of the functionality of befake at https://toofake.vercel.app/

**Comment by Damokoez** [comment]
that's amazing thank you! (thanks also for doing it for free! you are the best)Le mer. 21 juin 2023 à 23:06, s-alad ***@***.***> a écrit :
Hey, I fixed my web client that allows you to use most of the functionality of befake at https://toofake.vercel.app/

**Comment by s-alad** [comment]
No Problem! Let me know of any issues.
